doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1577,62 +1615,188 @@ Response code:
          }
         ]
 
-### updateContext [POST /v2/updateContext]
+### Register [POST /v2/op/register]
 
+This operation allows to create, update and/or delete several registrations in a single batch operation.
 The payload is an object with two properties:
 
-* `action`, to specify the kind of updateContext action to do (see OMA NGSI specification for the semantics),
-   either UPDATE, APPEND or DELETE
-* `entities`, an array of entities, each one specified using the the JSON entity representation format
-  (described in a section above).
+* `actionType`, to specify the kind of register action to do: either APPEND, UPDATE, DELETE.
```

Comment:
Regarding APPEND vs CREATE, the idea was to keep keywords similar to the ones used by updateContext, but maybe is a good idea to use different ones. I don't have a strong opinion in that really...